runtime/runtime/src/tests/apply.rs

```diff
@@ -86,20 +119,22 @@ fn setup_runtime_for_shard(
     let runtime = Runtime::new();
     let mut signers = vec![];
     let mut initial_state = tries.new_trie_update(shard_uid, root);
-    for account_id in initial_accounts.into_iter() {
-        let signer: Arc<Signer> = Arc::new(InMemorySigner::test_signer(&account_id));
+    for (account_id, signers_for_account) in accounts_with_keys.into_iter() {
```

Comment:
```suggestion
    for (account_id, signers_for_account) in accounts_with_keys {
```

the `into_iter` is inserted by the compiler.